specification/cognitiveservices/data-plane/VideoAnalysis/videoDescription.tsp

```diff
@@ -0,0 +1,385 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "./shared.tsp";
+
+using Azure.Core;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using OpenAPI;
+using Azure.ClientGenerator.Core;
+
+namespace VideoAnalysis;
+
+@pattern("^[a-z0-9-_]+$")
+@maxLength(32)
+@minLength(1)
+scalar TaskName extends string;
+
+@doc("The ApiVersion query parameter for VideoAnalysisTask API.")
+model VideoAnalysisTaskApiVersionParameter {
+    @doc("The API version to use for this operation.")
+    @query("api-version")
+    @minLength(1)
+    apiVersion?: string;
+}
+
+@doc("Time range considering the offset from 0.0s and lasting a duration.")
+model TimeRange {
+    @doc("Offset in ISO 8601 duration format, i.e., P[n]Y[n]M[n]DT[n]H[n]M[n]S or P[n]W.")
+    offset: string;
+
+    @doc("Duration in ISO 8601 duration format, i.e., P[n]Y[n]M[n]DT[n]H[n]M[n]S or P[n]W.")
+    duration?: string;
+}
+
+@doc("Authentication method to be used.")
+@discriminator("kind")
+model AuthenticationModel {
+    @doc("Type of authentication used")
+    kind: AuthenticationModelType;
+}
+
+@doc("No authentication model")
+model NoneAuthenticationModel extends AuthenticationModel {
+    kind: AuthenticationModelType.none;
+}
+
+@doc("Key based authentication model")
+model KeyAuthenticationModel extends AuthenticationModel {
+    kind: AuthenticationModelType.key;
+
+    @doc("Key used to authenticate.")
+    key: string;
+}
+
+@doc("Azure Managed Identity authentication model")
+model ManagedIdentityAuthenticationModel extends AuthenticationModel {
+    kind: AuthenticationModelType.managedIdentity;
+}
+
+@doc("Represents the Video Analysis Task response model for the JSON document. Returned after the creation of a video analysis task.")
+@resource("videodescriptions")
+model VideoAnalysisTaskResponseModel {
```

Comment:
We discourage "Response" and "Model" in model names.
```suggestion
model VideoAnalysisTask {
```

This applies to all the model names, but I will only comment here.